.github/workflows/Dockerfile.gnu

```diff
@@ -0,0 +1,68 @@
+#***********************************************************************
```

Comment:
should .github be updated to that in mixedmode?

Reply:
this may have been me messing up the mixedmode merge

Reply:
@mcallic2 this